src/ert/resources/workflows/jobs/internal-gui/scripts/gen_data_rft_export.py

```diff
@@ -244,12 +238,21 @@ def getArguments(self, parent, storage):
         success = dialog.showAndTell()
 
         if success:
-            ensemble_list = ",".join(list_edit.getItems())
+            ensemble_list = {
```

Comment:
just nitpicking here :) but should be `ensemble_dict`

Reply:
Thanks, using `ensemble_data_as_dict`.